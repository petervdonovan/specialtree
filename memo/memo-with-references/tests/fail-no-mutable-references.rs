use memo_with_references::memo;

#[memo('a)]
fn memome<'a>(data: &'a mut String) {}

fn main() {}
