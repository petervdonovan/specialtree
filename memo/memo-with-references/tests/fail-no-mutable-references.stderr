error: memoizable functions cannot have mutable reference parameters
 --> tests/fail-no-mutable-references.rs:4:21
  |
4 | fn memome<'a>(data: &'a mut String) {}
  |                     ^^^^^^^^^^^^^^
